src/cluster/managers/documentation/CommandDocumentationManager.ts

```diff
@@ -88,6 +88,31 @@ export class CommandDocumentationManager extends DocumentationTreeManager {
         };
     }
 
+    #listCommandNames(names: readonly string[]): string {
+        const charLimit = discord.getLimit('embed.field.value') - 6;
+        if (names.reduce((p, c) => p + c.length + 2, 0) - 2 <= charLimit)
+            return `\`\`\`${names.join(', ')}\`\`\``;
+
+        const remaining = [...names];
+        let missing = 0;
+        const postfix1 = '+ ';
+        const postfix2 = ' more';
+        function contentLength(): number {
+            return remaining.reduce((p, c) => p + c.length + 2, 0) - 2
+                + postfix1.length
+                + postfix2.length
+                + missing.toString().length;
```

Comment:
not a big thing but I would have placed the missing line before the postfix2 even though it does not change the value returned by contentLength()

Reply:
Fair point!